src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -91,35 +92,26 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
 
             var allSettings = _shellHost.GetAllSettings().OrderBy(s => s.Name);
             var dataProtector = _dataProtectorProvider.CreateProtector("Tokens").ToTimeLimitedDataProtector();
-
-            var pageSize = 10;
-            var siteService = HttpContext.RequestServices.GetService<ISiteService>();
-            if (siteService != null)
-            {
-                var siteSettings = await siteService.GetSiteSettingsAsync();
-                pageSize = siteSettings.PageSize;
-            }
-
-            var pager = new Pager(pagerParameters, pageSize);
+            var pager = new Pager(pagerParameters, _pagerOptions.PageSize);
 
             var entries = allSettings.Select(x =>
+            {
+                var entry = new ShellSettingsEntry
+                {
+                    Category = x["Category"],
+                    Description = x["Description"],
+                    Name = x.Name,
+                    ShellSettings = x,
+                    IsDefaultTenant = String.Equals(x.Name, ShellHelper.DefaultShellName, StringComparison.OrdinalIgnoreCase)
+                };
+
+                if (x.State == TenantState.Uninitialized && !String.IsNullOrEmpty(x["Secret"]))
                 {
-                    var entry = new ShellSettingsEntry
```

Comment:
I am not sure what happened here. I’ll revert some of these changes

Reply:
No problem, you formatted it like this which is good

            var entries = allSettings.Select(x =>
            {
                var entry = new ShellSettingsEntry
                {
                    Category = x["Category"],

While it was formatted like this

            var entries = allSettings.Select(x =>
                {
                    var entry = new ShellSettingsEntry
                    {
                        Category = x["Category"],

Reply:
As a side note, not now but here it will be `IsDefaultTenant = x.Name == ShellHelper.DefaultShellName`.

I think I will merge soon #12188 as there are too many PRs related to the same files since a too long time.

So that we will have the same base on which we will be able to add new things, even if it creates some PR conflicts.

Reply:
I fixed the conflict.